packages/code-connect/components/CodeBlock/CodeBlock.figma.tsx

```diff
@@ -0,0 +1,31 @@
+import figma from '@figma/code-connect';
+import { CodeBlock, ExpandableSectionToggle } from '@patternfly/react-core';
+
+figma.connect(
+  CodeBlock,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=9802-5857',
+  {
+    props: {
+      // TODO: FIGMA: configure options for actions
+      actions: `action1, action2, action3, action4, action5`,
+
+      // TODO: FIGMA: configure CodeBlockCode
+      expandableSection: figma.boolean('Expandable', {
+        true: (
+          <ExpandableSectionToggle ontoggle={() => {}} contentId="code-block-id" direction="up">
+            {isExpanded ? 'Show Less' : 'Show More'}
```

Comment:
in this case, isExpanded is not defined so they will get an `ReferenceError: isExpanded is not defined`